webapp/packages/core-ui/src/Form/FormPart.ts

```diff
@@ -17,6 +17,7 @@ export abstract class FormPart<TPartState, TFormState = any> implements IFormPar
   state: TPartState;
   initialState: TPartState;
   isSaving: boolean;
+  saved: boolean;
```

Comment:
Why do we need this flag? Our design was based on comparing the initial and current states, so the initial always needs to be synchronized with the server. When the initial state and current state are equal, then the  form is saved

How is this flag managed? What if the form gets a new initial state, and the current state will not equal the initial state? Will the form be marked as saved?

Reply:
removed